app/templates/buckets/list.html

```diff
@@ -1,31 +1,37 @@
 {% extends "layouts/one-column.html" %}
+{% from "includes/yesno.html" import yes_no %}
 
-{% set page_title = "App data sources" %}
+{% set page_title = "Data sources" %}
 {% set breadcrumbs = [{text: 'Home', url: '/'}, {text: 'List all data sources'}] %}
 
 {% block main_column %}
 
-<p>{{ buckets.length }} app data source{%- if buckets.length != 1 -%}s{%- endif -%}</p>
+<p>{{ buckets.length }} data source{%- if buckets.length != 1 -%}s{%- endif -%}</p>
+<!-- <p>{{ warehouse_buckets.length }} warehouse data source{%- if warehouse_buckets.length != 1 -%}s{%- endif -%}</p>
+<p>{{ app_buckets.length }} app data source{%- if app_buckets.length != 1 -%}s{%- endif -%}</p> -->
```

Comment:
Commented out code should be removed. Also, when putting comments in templates, consider using template comment tags, eg: `{# comment #}`

Reply:
👍 Forgot. Ta.